frontend/js/src/user-feed/UserFeed.tsx

```diff
@@ -487,9 +591,40 @@ export default function UserFeedPage() {
         </div>
       );
     }
+    if (event.event_type === EventType.THANKS && !event.hidden) {
+      const { metadata } = event as TimelineEvent<ThanksMetadata>;
+      return (
+        <div className="event-content">
+          <Card className="listen-card">
+            <div className="main-content">{metadata?.blurb_content}</div>
+          </Card>
+        </div>
+      );
+    }
     return null;
   };
 
+  const renderSubEvent = (subEvent: TimelineEvent<EventMetadata> | undefined) => {
```

Comment:
This function needs to be moved under the `renderEventText` function, I am getting a warning "'renderEventText' was used before it was defined"